app/v1/views/questions.py

```diff
@@ -2,48 +2,63 @@
 
 from flask import Blueprint, jsonify, request
 
+from app.utilities import check_keys, check_empty_dict
+from app.v1.models import Question
+
 questions = Blueprint("questions_blueprint", __name__,
-                      url_prefix="/stackoverflowlite/api/v1/questions")
+                      url_prefix="/stackoverflowlite/api/v1")
+
+question_instance = Question()
 
 
-@questions.route('/', methods=['POST'])
+@questions.route('/questions', methods=['POST'])
 def post_question():
     """method to post a new question"""
     question_info = request.get_json()
 
-    return jsonify({"Question": question_info})
-
-
-@questions.route('/', methods=['GET'])
+    # Check that all fields have been provided
+    if check_keys(question_info, 2):
+        response = jsonify(
+            {"Error": "Please provide question title and description"})
+        return response, 400
+
+    # Check for empty values
+    if check_empty_dict(question_info):
+        response = jsonify({"Error": "Fields must not be empty"})
+        return response, 400
+
+    # Check if question's already been asked
```

Comment:
Abstract this to a question title validation function.

Reply:
Done.